src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,196 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_checkNullUser_throwException() {
+        //Given
+        User user = null;
+        Storage.people.add(user);
+        //when
+        //then
+        Assertions.assertThrows(ValidationException.class,
+                () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_nonExisted_Ok() {
+        // Given
+        User user = new User();
+        user.setPassword("password1");
+        user.setLogin("loginuser");
+        user.setAge(30);
+        // When
+        User registeredUser = registrationService.register(user);
+        //then
+        Assertions.assertEquals(user.getAge(), registeredUser.getAge());
+        Assertions.assertEquals(user.getPassword(), registeredUser.getPassword());
+        Assertions.assertEquals(user.getLogin(), registeredUser.getLogin());
+        Assertions.assertNotNull(registeredUser.getId());
```

Comment:
Please use static imports for assert methods to make code cleaner
```suggestion
        assertEquals(user.getAge(), registeredUser.getAge());
        assertEquals(user.getPassword(), registeredUser.getPassword());
        assertEquals(user.getLogin(), registeredUser.getLogin());
        assertNotNull(registeredUser.getId());
```